packages/httpsnippet-client-api/src/index.js

```diff
@@ -109,9 +98,10 @@ module.exports = function (source, options) {
   code.push(`const sdk = require('api')('${opts.apiDefinitionUri}');`);
   code.blank();
 
-  // If we have multiple servers configured and our source URL differs from the stock URL that we receive from our
-  // `oas` library then the URL either has server variables contained in it (that don't match the defaults), or the
-  // OAS offers alternate server URLs and we should expose that in the generated snippet.
+  // If we have multiple servers configured and our source URL differs from the stock URL that we
+  // receive from our `oas` library then the URL either has server variables contained in it (that
+  // don't match the defaults), or the OAS offers alternate server URLs and we should expose that
+  // in the generated snippet.
```

Comment:
I've been trying to be a better small-screen citizen lately by capping my comments at the 100 character mark (traffic light yellow†) instead of going to a full 120. Hopefully you don't mind.

---
† traffic light yellow
```json
"[javascript]": {
  "editor.rulers": [
    80,
    { "column": 100, "color": "#F7B500" }, // traffic light yellow
    { "column": 120, "color": "#AA3F44" } // traffic light red
  ],
  "editor.tabSize": 2,
  "editor.defaultFormatter": "esbenp.prettier-vscode"
},
```

Reply:
thoughts on indenting continued lines? I've always found it hard to know where one thought ends and another starts without it.


e.g.

```
  // If we have multiple servers configured and our source URL differs from the stock URL that we
  //     receive from our `oas` library then the URL either has server variables contained in it (that
  //     don't match the defaults), or the OAS offers alternate server URLs and we should expose that
  //     in the generated snippet.
```

Reply:
also would this be better as a /* */ block? do we have an eslint saying when to use one or the other?

Reply:
I'm 1000% against indenting comments like this. I've also never personally liked using the `/* */` style of comment on comments that aren't a traditional method or class docblock. There's the [multiline-comment-style](https://eslint.org/docs/rules/multiline-comment-style) ESLint rule but it have any configuration for method/class docblock vs regular comments.